syntax = "proto3";

package kubetee;

import "attestation.proto";
import "aecs.proto";

option java_package = "com.alipay.kubetee.aecs.proto.v1";

message KubeConfig {
  string name = 1;
  string identity_key = 2;
  string client_rpc_secure = 3;
  string client_key = 4;
  string client_cert = 5;
  string client_ca = 6;
  string server_endpoint = 7;
  UnifiedAttestationPolicy server_policy = 8;
}

message SecretsParseResult {
  repeated EnclaveSecret secrets = 1;
}
